pypeit/core/meta.py

```diff
@@ -140,10 +140,10 @@ def define_additional_meta(nlamps=20):
                        'object': dict(dtype=str, comment='Alternative object name (cf. target)'),
                        'obstime': dict(dtype=str, comment='Observation time'),
                        'oscansec': dict(dtype=str, comment='Overscan section (windowing)'),
-                       'pressure': dict(dtype=float, comment='Pressure at obstime'),
+                       'pressure': dict(dtype=units.quantity.Quantity, comment='Pressure at obstime'),
```

Comment:
I try to avoid using astropy Quantities because they make numpy operations 10-100 times slower. An alternative here would be a float and you specify the units in the comments. As long as you are not multiplying this into an array and making that a quantity though it probably okay as is.

Reply:
OK - this is only used for the DAR correction, so I'm happy to return this to float.

Reply:
I don't feel strongly about it as long as you are not accidentally casting an array to a quantity. That said, if you don't use the quantity, please specify the units in the comment.